src/TLuaInterpreter.cpp

```diff
@@ -52,6 +52,17 @@
 #include <QRegularExpression>
 #include "post_guard.h"
 
+#include <QRegularExpression>
+#include <QSound>
+#include <QSslConfiguration>
+#include <QString>
+#include <QStringList>
+#include <QStringBuilder>
```

Comment:
Is this needed for this PR?

Reply:
No clue where the QSslConfiguration is coming from. I don't remember adding it, heh. I'll see what I can remove without breaking things.